updating/updating-cluster.adoc

```diff
@@ -7,6 +7,18 @@ toc::[]
 
 You can update, or upgrade, an {product-title} cluster.
 
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited:
+
+* You cannot update between different beta drops, such as from OpenShift 4 Beta
+Drop 2 to OpenShift 4 Beta Drop 3.
+* The ability to update from a beta drop of {product-title} to a full version of
+is not guaranteed.
+* After you install OpenShift 4 Beta Drop 3, you can update your cluster to the
+available nightly builds.
```

Comment:
"cluster" is singular, but "builds" is plural. Suggest changing it to "you can update your cluster to one of the
available nightly builds."